iterations_runners/iterations_runner.java

```diff
@@ -29,9 +29,18 @@ private String getCompJSON() {
     }
 
 
-    // Get GC info for the specified collector
+    /**
+     * Get GC info for the specified collector bean.
+     *
+     * @param gcBean Collector in question
+     * @return string of the form [name, [pools, time, count]]
+     */
     private String getGCJSON(GarbageCollectorMXBean gcBean) {
-        String json = "[";
+        /*
+         * Note, resisted the temptation to use a mapping from name
```

Comment:
Since this is inside a function, surely this would be neater with the `//` syntax?